lib/make-disk-image.nix

```diff
@@ -140,7 +161,27 @@ in
       done
       ${installer}
     ''}
-    export QEMU_OPTS=${lib.escapeShellArg "${QEMU_OPTS} -m 1024"}
+
+    if [[ -z $build_memory ]]; then
+      build_memory=1024
+    fi
+    if [[ -n $disk ]] && [ ${diskNumber} -eq 1 ]; then
+      export preVM=${diskoLib.writeCheckedBash { inherit pkgs checked; } "preVM.sh" ''
+        set -efu
+        mv copy_before_disko copy_after_disko xchg/
+      ''}
+      export postVM=${diskoLib.writeCheckedBash { inherit pkgs checked; } "postVM.sh" extraPostVM}
```

Comment:
usually the extraPostVM is used to compress the image. so not sure we would need to pass it in this case

Reply:
Ah I don't know, I didn't have a use case for it yet but in general it doesn't seem like it would hurt leaving it in here and would ensure that you have the same feature set.

Reply:
I finally came around to update the pr, removed the `--write-to-file` option and will do another pr adding it back in with the suggestions from this one once I find time for that.